chapter06/drjt/ex1.pl

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/perl
+use 5.010;
+use warnings;
+use strict;
+use Storable qw( nstore retrieve );
+
+my $storage = 'coconet.archive';
+
+my %total_bytes;
+if ( -r $storage ) {
```

Comment:
yeah, it would probably have been better to just do a `-f` test there, and just die if it runs into a permissions problem.